src/connection.h

```diff
@@ -124,6 +124,9 @@ typedef struct ConnectionType {
 
     /* Miscellaneous */
     int (*connIntegrityChecked)(void); // return 1 if connection type has built-in integrity checks
+
+    /* Get peer username based on connection type */
+    sds (*get_peer_username)(connection *conn);
```

Comment:
Let's put this next to `get_peer_cert` a few lines above

```c
    /* TLS specified methods */
    sds (*get_peer_cert)(struct connection *conn);
    sds (*get_peer_username)(connection *conn);
```

Reply:
Done updated